Source/Core/DolphinQt2/Settings.cpp

```diff
@@ -0,0 +1,71 @@
+// Copyright 2015 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include <QSize>
+
+#include "Common/FileUtil.h"
+#include "DolphinQt2/Settings.h"
+
+static QString GetSettingsPath()
+{
+	return QString::fromStdString(File::GetUserPath(D_CONFIG_IDX)) + QStringLiteral("/UI.ini");
+}
+
+Settings::Settings(QObject* parent)
+	: QSettings(GetSettingsPath(), QSettings::IniFormat, parent)
+{
+}
+
+QString Settings::GetThemeDir() const
+{
+	QString theme_name = value(QStringLiteral("Theme"), QStringLiteral("Clean")).toString();
+	return QString::fromStdString(File::GetThemeDir(theme_name.toStdString()));
+}
+
+QString Settings::GetLastGame() const
+{
+	return value(QStringLiteral("GameList/LastGame")).toString();
+}
+
+void Settings::SetLastGame(QString path)
+{
+	setValue(QStringLiteral("GameList/LastGame"), path);
+}
+
+QStringList Settings::GetPaths() const
+{
+	return value(QStringLiteral("GameList/Paths")).toStringList();
+}
+
+void Settings::SetPaths(QStringList paths)
+{
+	setValue(QStringLiteral("GameList/Paths"), paths);
+}
+
+DiscIO::IVolume::ELanguage Settings::GetDefaultGameLanguage() const
+{
+	return (DiscIO::IVolume::ELanguage) value(
+				QStringLiteral("GameList/GameLanguage"),
+				DiscIO::IVolume::LANGUAGE_ENGLISH).toInt();
+}
```

Comment:
Right, thanks for bringing this up. Is it reasonable to only use two languages, rather than three? That is, when we set the console language it sets both Wii and GC language in the Core.

If not, that's fine and I'll fix this to use SConfig's languages.